opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -43,6 +48,10 @@ message InstrumentationLibraryLogs {
 
   // A list of log records.
   repeated LogRecord logs = 2;
+
+  // This schema_url applies to all logs in the "logs" field regardless of the
+  // value of the schema_url field in the outer ResourceLogs message.
+  string schema_url = 3;
```

Comment:
How does this impact the API? Why is it not part of the `InstrumentationLibrary`?

Reply:
The API impact is shown here https://github.com/open-telemetry/opentelemetry-specification/pull/1666

I would prefer to keep `schema_url` inside InstrumentationLibrarySpans since it is not a property of the instrumentation library, it is a “meta” field which describes other fields in InstrumentationLibrarySpans and I think putting it inside InstrumentationLibrary is obscure. It would also look a bit strange to use a field in message A (InstrumentationLibrary) to describe fields in message B (repeated Span), when A and B are siblings and do not have parent-child relationship.

Reply:
Discussed with @bogdandrutu one more time and I believe what we have is the right approach. Resolving.